### PR TITLE
Add stable version of package coq-unimath

### PR DESCRIPTION
**Pull request by maggesi**

### CODE REVIEW

**Comment by palmskog** [comment]
@maggesi the layout document for the `released` repository states:
> The repository contains packages for Coq and for Coq extensions that were officially released by the Coq team or their corresponding authors.

I see no evidence that the commit the package uses was in any way officially released by the UniMath team. In particular, the version name of the package has no obvious connection to the commit.

Hence, my interpretation is that this package is not allowed in the `released` repo unless you make an official release (tarball) on GitHub or some other stable location. @gares what's your opinion?

**Comment by benediktahrens** [comment]
@palmskog : thanks a lot for your explanation. @dangrayson and I will make an official release of UniMath. Is it sufficient to use Github's "release" feature, which is, if I understand correctly, a git tag with some meta-information?

**Comment by palmskog** [comment]
@benediktahrens GitHub's release mechanism is essentially a thin layer on top of git tags, so it would be enough for the present purposes if you make a tag in the UniMath repository, and GitHub will give you a tarball I can use (see, e.g., https://github.com/coq-community/aac-tactics/tags). However, I recommend using the GitHub release feature to create the tag for you, since it gives some bells and whistles right away, such as nice link to the release on your GitHub repository page.

File: released/packages/coq-unimath/coq-unimath.20210805/opam [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+authors: ["The UniMath Development Team"]
+build: [make "BUILD_COQ=no" "-j%{jobs}%"]
+install: [make "BUILD_COQ=no" "install"]
+depends: [
+  "ocaml"
+  "coq" {>= "8.12.2"}
+]
+synopsis: "Library of Univalent Mathematics"
+url {
+  src: "git+https://github.com/UniMath/UniMath.git#7cc41b88e798e9f42af984d8a8abd72c5f955213"
```
Comment (gares) [inline released/packages/coq-unimath/coq-unimath.20210805/opam:16]: Please give a name to this hash

**Review by gares — CHANGES_REQUESTED** [review]

**Comment by benediktahrens** [comment]
I have made release 20210807, see https://github.com/UniMath/UniMath/releases/tag/v20210807. 
@maggesi : could you update the PR, please? It requires renaming the directory, I am afraid, as well as updating the source in the opam file. 
@palmskog : what is the policy on cryptographic integrity checking? Should a hash be included?

Thanks a lot for your help, @palmskog , @gares , @maggesi !

**Comment by palmskog** [comment]
@maggesi here is what I recommend that you use from line 15 in the file:
```
url {
  src: "https://github.com/UniMath/UniMath/archive/refs/tags/v20210807.tar.gz"
  checksum: "sha512=db2f660aea7e4c7eda6d10173173b80d357f994ff148102df6d20b0098e4c4e3ec17f9956700e118b7ff043a4ef29827cccd1fe9f5f54b5f5deaebcac60c666c"
}
```

File: released/packages/coq-unimath/coq-unimath.20210805/opam [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+}
```
Comment (gares) [inline released/packages/coq-unimath/coq-unimath.20210805/opam:17]:
```suggestion
url {
  src: "https://github.com/UniMath/UniMath/archive/refs/tags/v20210807.tar.gz"
  checksum: "sha512=db2f660aea7e4c7eda6d10173173b80d357f994ff148102df6d20b0098e4c4e3ec17f9956700e118b7ff043a4ef29827cccd1fe9f5f54b5f5deaebcac60c666c"
}
```

**Review by gares — APPROVED** [review]

**Comment by benediktahrens** [comment]
I think the directory name still needs to be changed, the 5 should become a 7.

**Comment by maggesi** [comment]
@palmskog @gares : Thank you for the help.
Should be ready to merge now.

**Pull request merged**